services/explorer/src/pages/TransactionData/TransactionData.effects.tsx

```diff
@@ -24,9 +31,20 @@ export const useTransactionDataContainerEffects = (txHash: string) => {
     },
     pause: !txHash,
   });
+
+  const { backendEndpoint } = useExplorerNetwork();
+  const txType = getTransactionType(transactionData);
+  const { isFetching, data: decodedTxData } = useGetDecodedTransactionsByTxHash(
+    backendEndpoint,
+    txType as TransactionType,
+    txHash
+  );
   const [showLoadingSkeleton, setShowLoadingSkeleton] =
     useState<boolean>(false);
 
+  const [showLoadingDecodedSkeleton, setShowLoadingDecodedSkeleton] =
+    useState<boolean>(false);
+
```

Comment:
Don't need this. Just rename `isFetching` to `showLoadingDecodedSkeleton`

Reply:
We removed this in the refactors and the skeleton wasn't showing at all.